docker-compose.deploy.yml

```diff
@@ -924,6 +920,40 @@ services:
       options:
         gelf-address: 'udp://127.0.0.1:12201'
         tag: 'webhooks'
+
+  dashboards:
+    volumes:
+      - /data/metabase:/data/metabase
+    networks:
+      - overlay_net
+    environment:
+      # Note that only mongo1 is used here on purpose for all environments
+      # https://discourse.metabase.com/t/solved-add-mongodb-replicaset-database/7404/2
+      - OPENCRVS_METABASE_DB_HOST=mongo1
+      - OPENCRVS_METABASE_DB_USER=performance
+      - OPENCRVS_METABASE_DB_PASS=${PERFORMANCE_MONGODB_PASSWORD}
+      - OPENCRVS_METABASE_DB_AUTH_DB=performance
+      - OPENCRVS_METABASE_SITE_NAME=OpenCRVS Dashboards
+      - OPENCRVS_METABASE_SITE_URL=http://metabase.{{hostname}}
+    deploy:
+      labels:
+        - 'traefik.enable=true'
+        - 'traefik.http.routers.metabase.rule=Host(`metabase.{{hostname}}`)'
+        - 'traefik.http.services.metabase.loadbalancer.server.port=4444'
+        - 'traefik.http.routers.metabase.tls=true'
```

Comment:
@rikukissa we need to update [dev.sh](https://github.com/opencrvs/opencrvs-core/blob/3813d15e2d17066349b3aa8bd49193f0b32d50f0/dev.sh#L94) & [setup.sh](https://github.com/opencrvs/opencrvs-core/blob/3813d15e2d17066349b3aa8bd49193f0b32d50f0/setup.sh#L278) to include port 4444 to make sure it is open